static/templates/electionResult.hbs

```diff
@@ -1,12 +1,21 @@
 <div>
   {{#each this}}
-  <div class="election-result box">
-    <h4 style="color: {{this.color}}">{{this.electionName}}</h4>
-    <div>Next general election: {{this.electionDate}}</div>
+  <div class="result">
+    <h3 style="color: {{this.color}}"><a href="{{this.url}}">{{this.electionName}}</a></h3>
```

Comment:
After these changes, the per-district colors aren't showing up anymore. I think that's fine, since the design called for color icons instead of color text--I'll file a separate issue for that.